--- anaconda-11.4.1.27/loader2/loader.c~	2008-08-27 01:39:53.000000000 +0200
+++ anaconda-11.4.1.27/loader2/loader.c	2008-08-27 01:42:07.063163028 +0200
@@ -1136,7 +1136,7 @@
 
     char *url = NULL, *ret = NULL, *devName = NULL, *kbdtype = NULL;
     static struct networkDeviceConfig netDev;
-    int i, rc, dir = 1;
+    int i, rc = LOADER_NOOP, dir = 1;
     int needsNetwork = 0, class = -1;
     int skipMethodDialog = 0, skipLangKbd = 0;
 
--- anaconda-11.4.1.27/loader2/nfsinstall.c~	2008-08-11 21:02:14.000000000 +0200
+++ anaconda-11.4.1.27/loader2/nfsinstall.c	2008-08-27 01:48:04.672161030 +0200
@@ -58,7 +58,7 @@
     int rc;
 
     entries[0].text = _("NFS server name:");
-    entries[0].value = &newServer;
+    entries[0].value = (const char **)(&newServer);
     entries[0].flags = NEWT_FLAG_SCROLL;
 
     if (asprintf(&entries[1].text, _("%s directory:"),
@@ -67,7 +67,7 @@
         abort();
     }
 
-    entries[1].value = &newDir;
+    entries[1].value = (const char **)(&newDir);
     entries[1].flags = NEWT_FLAG_SCROLL;
     entries[2].text = NULL;
     entries[2].value = NULL;
--- anaconda-11.4.1.27/loader2/net.c~	2008-08-11 21:02:14.000000000 +0200
+++ anaconda-11.4.1.27/loader2/net.c	2008-08-27 01:50:47.849163064 +0200
@@ -290,8 +290,8 @@
     }
 
     do {
-        struct newtWinEntry entry[] = { { N_("ESSID"), &essid, 0 },
-                                        { N_("Encryption Key"), &wepkey, 0 },
+        struct newtWinEntry entry[] = { { N_("ESSID"), (const char **)(&essid), 0 },
+                                        { N_("Encryption Key"), (const char **)(&wepkey), 0 },
                                         { NULL, NULL, 0 } };
 
         rc = newtWinEntries(_("Wireless Settings"), buf,
@@ -329,7 +329,7 @@
     struct in_addr addr;
     struct in6_addr addr6;
     char * ns = "";
-    struct newtWinEntry entry[] = { { N_("Nameserver IP"), &ns, 0 },
+    struct newtWinEntry entry[] = { { N_("Nameserver IP"), (const char **)(&ns), 0 },
                                       { NULL, NULL, 0 } };
 
     do {
@@ -1851,7 +1851,7 @@
     char **deviceNames;
     char *ksMacAddr = NULL, *seconds = strdup("10"), *idstr = NULL;
     struct device **devs;
-    struct newtWinEntry entry[] = {{N_("Seconds:"), (char **) &seconds, 0},
+    struct newtWinEntry entry[] = {{N_("Seconds:"), (const char **) &seconds, 0},
                                    {NULL, NULL, 0 }};
 
     devs = getDevices(DEVICE_NETWORK);
--- anaconda-11.4.1.27/yuminstall.py~	2008-08-11 21:02:14.000000000 +0200
+++ anaconda-11.4.1.27/yuminstall.py	2008-08-27 01:50:47.849163064 +0200
@@ -1673,8 +1673,6 @@
                 anaconda.id.desktop.setDefaultRunLevel(5)
                 break
 
-        for repo in self.ayum.repos.listEnabled():
-            repo.dirCleanup()
 
         # expire yum caches on upgrade
         if anaconda.id.getUpgrade() and os.path.exists("%s/var/cache/yum" %(anaconda.rootPath,)):
