lazy to update llh, just disable xen for now

--- anaconda-11.0.5/loader2/hardware.c~	2006-02-06 23:32:13.000000000 +0200
+++ anaconda-11.0.5/loader2/hardware.c	2006-05-17 00:22:14.317066126 +0300
@@ -67,7 +67,7 @@
     
     devices = probeDevices(CLASS_UNSPEC,
                            BUS_PCI | BUS_SBUS | BUS_VIO | BUS_MACIO |
-                           BUS_PCMCIA | BUS_XEN,
+                           BUS_PCMCIA,
                            PROBE_ALL);
 
     logMessage(DEBUGLVL, "finished bus probing");
