--- ./scripts/pkgorder	2006-05-18 00:09:50.134726080 +0300
+++ /pkgorder	2006-05-18 00:09:51.774762776 +0300
@@ -166,30 +166,30 @@
     ds = PackageOrderer(arch=arch)
     ds.setup(fn=config, excludes=options.excludeList, root = testpath)
 
-    addGroups(ds, ["core", "base", "text-internet"])
+#    addGroups(ds, ["core", "base", "text-internet"])
 
     # hack, hack, hack... make sure iscsi ends up on disc1 (#208832)
-    printMatchingPkgs("iscsi-*")        
+#    printMatchingPkgs("iscsi-*")        
 
-    addGroups(ds, ["base-x", "dial-up",
-                   "graphical-internet", "editors", 
-                   "graphics", "gnome-desktop", "sound-and-video",
-                   "printing"])
-
-    addGroups(ds, ["office", "engineering-and-scientific",
-                   "authoring-and-publishing", "games"])
-
-    addGroups(ds, ["web-server", "ftp-server", "sql-server",
-                   "mysql", "server-cfg", "dns-server",
-                   "smb-server", "admin-tools"])
-
-    addGroups(ds, ["kde-desktop", "development-tools", "development-libs",
-                   "gnome-software-development", "eclipse",
-                   "x-software-development",
-                   "java-development", "kde-software-development",
-                   "mail-server", "network-server", "legacy-network-server"])
+#    addGroups(ds, ["base-x", "dial-up",
+#                   "graphical-internet", "editors", 
+#                   "graphics", "gnome-desktop", "sound-and-video",
+#                   "printing"])
+
+#    addGroups(ds, ["office", "engineering-and-scientific",
+#                   "authoring-and-publishing", "games"])
+
+#    addGroups(ds, ["web-server", "ftp-server", "sql-server",
+#                   "mysql", "server-cfg", "dns-server",
+#                   "smb-server", "admin-tools"])
+
+#    addGroups(ds, ["kde-desktop", "development-tools", "development-libs",
+#                   "gnome-software-development", "eclipse",
+#                   "x-software-development",
+#                   "java-development", "kde-software-development",
+#                   "mail-server", "network-server", "legacy-network-server"])
 
-    addGroups(ds, ["news-server", "legacy-software-development"])
+#    addGroups(ds, ["news-server", "legacy-software-development"])
 
     #Everthing else but kernels
     for po in ds.pkgSack.returnPackages():
