--- anaconda-11.2.0.24.org/yuminstall.py	2007-02-13 14:56:08.000000000 +0100
+++ anaconda-11.2.0.24/yuminstall.py	2007-02-20 19:12:05.894623000 +0100
@@ -1009,7 +1009,7 @@
 
         # FIXME: this is a bit of a hack.  we shouldn't hard-code and
         # instead check by provides.  but alas.
-        for k in ("kernel", "kernel-smp", "kernel-xen0", "kernel-xen"):
+        for k in ("kernel", "kernel-smp", "kernel-grsecurity", "kernel-grsecurity-smp", "kernel-xen0", "kernel-xen"):
             if len(self.ayum.tsInfo.matchNaevr(name=k)) > 0:            
                 foundkernel = True
 
