--- anaconda-11.4.0.8/loader2/loader.c~	2007-12-14 02:04:10.000000000 +0100
+++ anaconda-11.4.0.8/loader2/loader.c	2007-12-14 02:04:20.783481654 +0100
@@ -27,6 +27,7 @@
 #include <execinfo.h>
 #include <fcntl.h>
 #include <newt.h>
+#include "lang.h"
 #include <popt.h>
 #include <signal.h>
 #include <stdio.h>
@@ -54,7 +55,6 @@
 #include "loader.h"
 #include "loadermisc.h" /* JKFIXME: functions here should be split out */
 #include "log.h"
-#include "lang.h"
 #include "fwloader.h"
 #include "kbd.h"
 #include "kickstart.h"
--- anaconda-11.4.0.8/loader2/modules.c~	2007-12-11 00:09:05.000000000 +0100
+++ anaconda-11.4.0.8/loader2/modules.c	2007-12-14 02:14:35.241251371 +0100
@@ -21,6 +21,7 @@
 #include <fcntl.h>
 #include <kudzu/kudzu.h>
 #include <newt.h>
+#include "lang.h"
 #include <popt.h>
 #include <stdlib.h>
 #include <stdio.h>
--- anaconda-11.4.0.8/loader2/driverdisk.c~	2007-12-11 00:09:05.000000000 +0100
+++ anaconda-11.4.0.8/loader2/driverdisk.c	2007-12-14 02:16:02.235272469 +0100
@@ -17,6 +17,7 @@
 #include <fcntl.h>
 #include <kudzu/kudzu.h>
 #include <newt.h>
+#include "lang.h"
 #include <popt.h>
 #include <stdlib.h>
 #include <stdio.h>
@@ -28,7 +29,6 @@
 #include "loader.h"
 #include "log.h"
 #include "loadermisc.h"
-#include "lang.h"
 #include "fwloader.h"
 #include "method.h"
 #include "modules.h"
--- anaconda-11.4.0.8/loader2/kickstart.c~	2007-12-11 00:09:05.000000000 +0100
+++ anaconda-11.4.0.8/loader2/kickstart.c	2007-12-14 02:26:34.191839534 +0100
@@ -23,6 +23,7 @@
 #include <fcntl.h>
 #include <kudzu/kudzu.h>
 #include <newt.h>
+#include "lang.h"
 #include <popt.h>
 #include <stdlib.h>
 #include <string.h>
@@ -31,7 +32,6 @@
 
 #include "loader.h"
 #include "loadermisc.h"
-#include "lang.h"
 #include "log.h"
 #include "kickstart.h"
 
--- anaconda-11.4.0.8/loader2/hdinstall.c~	2007-12-11 00:09:05.000000000 +0100
+++ anaconda-11.4.0.8/loader2/hdinstall.c	2007-12-14 02:27:38.187441262 +0100
@@ -20,6 +20,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <newt.h>
+#include "lang.h"
 #include <popt.h>
 #include <stdlib.h>
 #include <string.h>
@@ -33,7 +34,6 @@
 #include "loader.h"
 #include "loadermisc.h"
 #include "log.h"
-#include "lang.h"
 #include "modules.h"
 #include "method.h"
 #include "mediacheck.h"
--- anaconda-11.4.0.8/loader2/nfsinstall.c~	2007-12-11 00:09:05.000000000 +0100
+++ anaconda-11.4.0.8/loader2/nfsinstall.c	2007-12-14 02:28:21.684451811 +0100
@@ -18,13 +18,13 @@
 
 #include <fcntl.h>
 #include <newt.h>
+#include "lang.h"
 #include <popt.h>
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
 
 #include "loader.h"
-#include "lang.h"
 #include "loadermisc.h"
 #include "kickstart.h"
 #include "log.h"
--- anaconda-11.4.0.8/loader2/urlinstall.c~	2007-12-11 00:09:05.000000000 +0100
+++ anaconda-11.4.0.8/loader2/urlinstall.c	2007-12-14 02:29:03.181599806 +0100
@@ -17,14 +17,15 @@
  */
 
 #include <newt.h>
-#include <popt.h>
 #include <stdlib.h>
 #include <string.h>
 #include <sys/mount.h>
 #include <unistd.h>
 
 #include "../isys/nl.h"
 
+#include "lang.h"
+#include <popt.h>
 #include "copy.h"
 #include "kickstart.h"
 #include "loader.h"
--- anaconda-11.4.0.8/loader2/net.c~	2007-12-11 00:09:05.000000000 +0100
+++ anaconda-11.4.0.8/loader2/net.c	2007-12-14 02:35:20.655656874 +0100
@@ -27,6 +27,7 @@
 #include <sys/utsname.h>
 #include <arpa/inet.h>
 #include <errno.h>
+#include "lang.h"
 #include <popt.h>
 #include <resolv.h>
 #include <net/if.h>
@@ -44,7 +45,6 @@
 #include "../isys/nl.h"
 #include "../isys/str.h"
 
-#include "lang.h"
 #include "loader.h"
 #include "loadermisc.h"
 #include "log.h"
