--- ./scripts/pkgorder	2006-05-18 00:09:50.134726080 +0300
+++ /pkgorder	2006-05-18 00:09:51.774762776 +0300
@@ -156,27 +156,27 @@
     ds = PackageOrderer(arch=arch)
     ds.setup(fn=config, excludes=options.excludeList, root = testpath)
 
-    addGroups(ds, ["Core", "Base", "Text-based Internet"])
-
-    addGroups(ds, ["X Window System", "Dialup Networking Support",
-                   "Graphical Internet", "Editors", 
-                   "Graphics", "GNOME Desktop Environment", "Sound and Video",
-                   "Printing Support", "Dialup Networking Support"])
-
-    addGroups(ds, ["Office/Productivity", "Engineering and Scientific",
-                   "Authoring and Publishing", "Games and Entertainment"])
-
-    addGroups(ds, ["Web Server", "FTP Server", "MySQL Database",
-                   "Server Configuration Tools", "DNS Name Server",
-                   "Windows File Server", "Administration Tools"])
-
-    addGroups(ds, ["KDE (K Desktop Environment)", "Development Tools",
-                   "GNOME Software Development", "Eclipse",
-                   "Java Development", "KDE Software Development",
-                   "PostgreSQL Database", "Mail Server",
-                   "Legacy Network Server"])
-
-    addGroups(ds, ["News Server", "Legacy Software Development"])
+#    addGroups(ds, ["Base", "Networking"])
+#
+#    addGroups(ds, ["X Window System", "Dialup Networking Support",
+#                   "Graphical Internet", "Editors", 
+#                   "Graphics", "GNOME Desktop Environment", "Sound and Video",
+#                   "Printing Support", "Dialup Networking Support"])
+#
+#    addGroups(ds, ["Office/Productivity", "Engineering and Scientific",
+#                   "Authoring and Publishing", "Games and Entertainment"])
+#
+#    addGroups(ds, ["Web Server", "FTP Server", "MySQL Database",
+#                   "Server Configuration Tools", "DNS Name Server",
+#                   "Windows File Server", "Administration Tools"])
+#
+#    addGroups(ds, ["KDE (K Desktop Environment)", "Development Tools",
+#                   "GNOME Software Development", "Eclipse",
+#                   "Java Development", "KDE Software Development",
+#                   "PostgreSQL Database", "Mail Server",
+#                   "Legacy Network Server"])
+#
+#    addGroups(ds, ["News Server", "Legacy Software Development"])
 
     #Everthing else but kernels - don't depsolve
     for po in ds.pkgSack.returnPackages():
