diff -ur anaconda-11.2.0.24.org/partedUtils.py anaconda-11.2.0.24/partedUtils.py
--- anaconda-11.2.0.24.org/partedUtils.py	2007-01-16 20:58:13.000000000 +0100
+++ anaconda-11.2.0.24/partedUtils.py	2007-02-20 18:33:58.522623000 +0100
@@ -472,8 +472,8 @@
     return None
 
 def getReleaseString(mountpoint):
-    if os.access(mountpoint + "/etc/redhat-release", os.R_OK):
-        f = open(mountpoint + "/etc/redhat-release", "r")
+    if os.access(mountpoint + "/etc/pld-release", os.R_OK):
+        f = open(mountpoint + "/etc/pld-release", "r")
         try:
             lines = f.readlines()
         except IOError:
@@ -490,22 +490,12 @@
 
         # get the release name and version
         # assumes that form is something
-        # like "Red Hat Linux release 6.2 (Zoot)"
-        if relstr.find("release") != -1:
-            try:
-                idx = relstr.find("release")
-                prod = relstr[:idx - 1]
-
-                ver = ""
-                for a in relstr[idx + 8:]:
-                    if a in string.digits + ".":
-                        ver = ver + a
-                    else:
-                        break
-
-                    relstr = prod + " " + ver
-            except:
-                pass # don't worry, just use the relstr as we have it
+	# like 2.99 PLD Linux (Th)
+	try:
+	    relstr = relstr.split(' ')
+	    relstr = relstr[0]
+	except:
+	    pass # don't worry, just use the relstr as we have it
         return relstr
     return ""
 
diff -ur anaconda-11.2.0.24.org/yuminstall.py anaconda-11.2.0.24/yuminstall.py
--- anaconda-11.2.0.24.org/yuminstall.py	2007-02-13 14:56:08.000000000 +0100
+++ anaconda-11.2.0.24/yuminstall.py	2007-02-20 18:26:50.814623000 +0100
@@ -1269,7 +1269,7 @@
         # Figure out current version for upgrade nag and for determining weird
         # upgrade cases
         supportedUpgradeVersion = -1
-        for pkgtup in self.ayum.rpmdb.whatProvides('redhat-release', None, None):
+        for pkgtup in self.ayum.rpmdb.whatProvides('issue', None, None):
             n, a, e, v, r = pkgtup
             if supportedUpgradeVersion <= 0:
                 val = rpmUtils.miscutils.compareEVR((None, '3', '1'),
