--- anaconda-11.0.5/yuminstall.py~	2006-05-24 02:51:56.715042751 +0300
+++ anaconda-11.0.5/yuminstall.py	2006-05-24 03:01:04.177356166 +0300
@@ -701,7 +701,7 @@
 
         # FIXME: this is a bit of a hack.  we shouldn't hard-code and
         # instead check by provides.  but alas.
-        for k in ("kernel", "kernel-smp", "kernel-xen0"):
+        for k in ("kernel-grsecurity", "kernel-grsecurity-smp", "kernel-xen0"):
             if len(self.ayum.tsInfo.matchNaevr(name=k)) > 0:            
                 foundkernel = True
 
@@ -736,8 +736,8 @@
 
         if not foundkernel and (isys.smpAvailable() or isys.htavailable()):
             try:
-                ksmp = getBestKernelByArch("kernel-smp", self.ayum)
-                log.info("selected kernel-smp package for kernel")
+                ksmp = getBestKernelByArch("kernel-grsecurity-smp", self.ayum)
+                log.info("selected kernel-grsecurity-smp package for kernel")
                 foundkernel = True
             except PackageSackError:
                 ksmp = None
@@ -1028,7 +1028,7 @@
         kernelVersions = []
 
         # nick is used to generate the lilo name
-        for (ktag, nick) in [ ('kernel-smp', 'smp'),
+        for (ktag, nick) in [ ('kernel-grsecurity-smp', 'smp'),
                               ('kernel-xen0', 'xen0'),
                               ('kernel-xenU', 'xenU') ]:
             tag = ktag.rsplit('-', 1)[1]
