diff -ur anaconda-11.2.0.24.org/partedUtils.py anaconda-11.2.0.24/partedUtils.py
--- anaconda-11.2.0.24.org/partedUtils.py	2007-01-16 20:58:13.000000000 +0100
+++ anaconda-11.2.0.24/partedUtils.py	2007-02-20 18:33:58.522623000 +0100
@@ -472,8 +472,8 @@
     return None
 
 def getReleaseString(mountpoint):
-    if os.access(mountpoint + "/etc/redhat-release", os.R_OK):
-        f = open(mountpoint + "/etc/redhat-release", "r")
+    if os.access(mountpoint + "/etc/pld-release", os.R_OK):
+        f = open(mountpoint + "/etc/pld-release", "r")
         try:
             lines = f.readlines()
         except IOError:
@@ -490,22 +490,12 @@
 
         # get the release name and version
         # assumes that form is something
-        # like "Red Hat Linux release 6.2 (Zoot)"
-        if relstr.find("release") != -1:
-            try:
-                idx = relstr.find("release")
-                prod = relstr[:idx - 1]
-
-                ver = ""
-                for a in relstr[idx + 8:]:
-                    if a in string.digits + ".":
-                        ver = ver + a
-                    else:
-                        break
-
-                    relstr = prod + " " + ver
-            except:
-                pass # don't worry, just use the relstr as we have it
+	# like 2.99 PLD Linux (Th)
+	try:
+	    relstr = relstr.split(' ')
+	    relstr = relstr[0]
+	except:
+	    pass # don't worry, just use the relstr as we have it
         return relstr
     return ""
 
diff -ur anaconda-11.2.0.24.org/scripts/pkgorder anaconda-11.2.0.24/scripts/pkgorder
--- anaconda-11.2.0.24.org/scripts/pkgorder	2007-02-20 18:23:45.538623000 +0100
+++ anaconda-11.2.0.24/scripts/pkgorder	2007-02-20 18:28:05.766623000 +0100
@@ -107,10 +107,10 @@
 def createConfig(toppath):
     yumconfstr = """
 [main]
-distroverpkg=redhat-release
+distroverpkg=issue
 gpgcheck=0
 reposdir=/dev/null
-exclude=*debuginfo*
+exclude=vserver-packages *debuginfo*
 
 [anaconda]
 name=Anaconda
diff -ur anaconda-11.2.0.24.org/scripts/splittree.py anaconda-11.2.0.24/scripts/splittree.py
--- anaconda-11.2.0.24.org/scripts/splittree.py	2006-12-13 16:55:23.000000000 +0100
+++ anaconda-11.2.0.24/scripts/splittree.py	2007-02-20 18:27:17.982623000 +0100
@@ -99,7 +99,7 @@
         self.src_list = []
         self.shared_list = []
         self.reverse_sort_srpms=None
-        self.common_files = ['beta_eula.txt', 'EULA', 'README', 'GPL', 'RPM-GPG-KEY', 'RPM-GPG-KEY-beta', 'RPM-GPG-KEY-fedora']
+        self.common_files = ['beta_eula.txt', 'EULA', 'README', 'GPL', 'RPM-GPG-KEY', 'RPM-GPG-KEY-beta', 'RPM-GPG-KEY-PLD']
         self.logfile = []
 
 
diff -ur anaconda-11.2.0.24.org/scripts/yumcache anaconda-11.2.0.24/scripts/yumcache
--- anaconda-11.2.0.24.org/scripts/yumcache	2007-01-30 22:55:56.000000000 +0100
+++ anaconda-11.2.0.24/scripts/yumcache	2007-02-20 18:24:43.130623000 +0100
@@ -24,7 +24,7 @@
 
         self.yumconfstr = """
 [main]
-distroverpkg=redhat-release
+distroverpkg=issue
 cachedir=%s
 gpgcheck=0
 
diff -ur anaconda-11.2.0.24.org/yuminstall.py anaconda-11.2.0.24/yuminstall.py
--- anaconda-11.2.0.24.org/yuminstall.py	2007-02-13 14:56:08.000000000 +0100
+++ anaconda-11.2.0.24/yuminstall.py	2007-02-20 18:26:50.814623000 +0100
@@ -1269,7 +1269,7 @@
         # Figure out current version for upgrade nag and for determining weird
         # upgrade cases
         supportedUpgradeVersion = -1
-        for pkgtup in self.ayum.rpmdb.whatProvides('redhat-release', None, None):
+        for pkgtup in self.ayum.rpmdb.whatProvides('issue', None, None):
             n, a, e, v, r = pkgtup
             if supportedUpgradeVersion <= 0:
                 val = rpmUtils.miscutils.compareEVR((None, '3', '1'),
