--- anaconda-10.2.0.52-old/scripts/getkeymaps	2004-05-20 03:00:03.000000000 +0200
+++ anaconda-10.2.0.52/scripts/getkeymaps	2005-04-24 16:34:23.401958968 +0200
@@ -37,7 +37,7 @@
 
 for map in $MAPS ; do 
  eval find /lib/kbd/keymaps/$PATTERN -name "$map.map*" | while read n; do
-    /bin/loadkeys `basename $n .gz` >/dev/null
+    /usr/bin/loadkeys `basename $n .gz` >/dev/null
     $READMAP $TMPDIR/`basename $n .map.gz`.map
   done
 done
--- anaconda-11.0.5/fsset.py~	2006-03-13 21:12:36.000000000 +0200
+++ anaconda-11.0.5/fsset.py	2006-05-24 19:47:33.500294489 +0300
@@ -203,7 +203,7 @@
             w = None
         
         devicePath = entry.device.setupDevice(chroot)
-        args = [ "/usr/sbin/badblocks", "-vv", devicePath ]
+        args = [ "/sbin/badblocks", "-vv", devicePath ]
 
         # entirely too much cutting and pasting from ext2FormatFileSystem
         fd = os.open("/dev/tty5", os.O_RDWR | os.O_CREAT | os.O_APPEND)
