--- anaconda/Makefile.inc.org	2007-02-20 21:29:51.230623000 +0100
+++ anaconda/Makefile.inc	2007-02-20 21:30:25.302623000 +0100
@@ -1,8 +1,8 @@
 #
 # define destination locations
 #
-PYTHONLIBDIR = /usr/lib/anaconda
-RUNTIMEDIR = /usr/lib/anaconda-runtime
+PYTHONLIBDIR = /usr/$(LIBDIR)/anaconda
+RUNTIMEDIR = /usr/$(LIBDIR)/anaconda-runtime
 ANACONDADATADIR = /usr/share/anaconda
 PYVER  := $(shell python -c 'import sys; print sys.version[0:3]')
 PYTHON = python$(PYVER)
diff -ur anaconda-11.2.0.24/anaconda anaconda-11.2.0.24/anaconda
--- anaconda-11.2.0.24/anaconda	2007-02-01 23:55:38.000000000 +0100
+++ anaconda-11.2.0.24/anaconda	2007-02-20 18:15:06.726623000 +0100
@@ -147,17 +142,7 @@
 			   "/tmp/updates/%s/%s" %(pypkg, f))
 
 	# get the libdir.  *sigh*
-	if os.access("/usr/lib64/python%s/site-packages/%s" %(pyver, pypkg),
-		     os.X_OK):
-	    libdir = "lib64"
-        elif os.access("/usr/lib/python%s/site-packages/%s" %(pyver, pypkg),
-                       os.X_OK):
-	    libdir = "lib"
-	else:
-            # If the directory doesn't exist, there's nothing to link over.
-            # This happens if we forgot to include one of the above packages
-            # in the image, for instance.
-            continue
+	libdir = sys.lib
 
 	if os.access("/tmp/updates/%s" %(pypkg,), os.X_OK):
 	    for f in os.listdir("/usr/%s/python%s/site-packages/%s" %(libdir,
@@ -303,15 +288,15 @@
         sys.path.append('textw')
         sys.path.append('iw')
     else:
-        sys.path.append('/usr/lib/anaconda')
-        sys.path.append('/usr/lib/anaconda/textw')
-        sys.path.append('/usr/lib/anaconda/iw')
+        sys.path.append('/usr/%s/anaconda' % sys.lib)
+        sys.path.append('/usr/%s/anaconda/textw' % sys.lib)
+        sys.path.append('/usr/%s/anaconda/iw' % sys.lib)
 
     if (os.path.exists('booty')):
         sys.path.append('booty')
         sys.path.append('booty/edd')
     else:
-        sys.path.append('/usr/lib/booty')
+        sys.path.append('/usr/%s/booty' % sys.lib)
 
     sys.path.append('/usr/share/system-config-date')
 
@@ -390,7 +375,7 @@
     if child == 0:
         for p in ('/mnt/source/RHupdates/pyrc.py', \
                 '/tmp/updates/pyrc.py', \
-                '/usr/lib/anaconda-runtime/pyrc.py'):
+                '/usr/%s/anaconda-runtime/pyrc.py' % sys.lib):
             if os.access(p, os.R_OK|os.X_OK):
                 os.environ['PYTHONSTARTUP'] = p
                 break
diff -ur anaconda-11.2.0.24/command-stubs/list-harddrives-stub anaconda-11.2.0.24/command-stubs/list-harddrives-stub
--- anaconda-11.2.0.24/command-stubs/list-harddrives-stub	2006-06-22 19:18:11.000000000 +0200
+++ anaconda-11.2.0.24/command-stubs/list-harddrives-stub	2007-02-20 18:19:21.454623000 +0100
@@ -10,7 +10,7 @@
 if (os.path.exists('isys')):
     sys.path.append('isys')
 
-sys.path.append('/usr/lib/anaconda')
+sys.path.append('/usr/%s/anaconda' % sys.lib)
 
 import anaconda_log
 import parted
diff -ur anaconda-11.2.0.24/command-stubs/loadkeys-stub anaconda-11.2.0.24/command-stubs/loadkeys-stub
--- anaconda-11.2.0.24/command-stubs/loadkeys-stub	2004-05-11 23:52:00.000000000 +0200
+++ anaconda-11.2.0.24/command-stubs/loadkeys-stub	2007-02-20 18:18:48.482623000 +0100
@@ -7,7 +7,7 @@
 if (os.path.exists('isys')):
     sys.path.append('isys')
 
-sys.path.append('/usr/lib/anaconda')
+sys.path.append('/usr/%s/anaconda' % sys.lib)
 
 import isys
 
diff -ur anaconda-11.2.0.24/command-stubs/losetup-stub anaconda-11.2.0.24/command-stubs/losetup-stub
--- anaconda-11.2.0.24/command-stubs/losetup-stub	2001-02-20 20:48:12.000000000 +0100
+++ anaconda-11.2.0.24/command-stubs/losetup-stub	2007-02-20 18:19:53.858623000 +0100
@@ -7,7 +7,7 @@
 if (os.path.exists('isys')):
     sys.path.append('isys')
 
-sys.path.append('/usr/lib/anaconda')
+sys.path.append('/usr/%s/anaconda' % sys.lib)
 
 import isys
 from sys import argv
diff -ur anaconda-11.2.0.24/command-stubs/mknod-stub anaconda-11.2.0.24/command-stubs/mknod-stub
--- anaconda-11.2.0.24/command-stubs/mknod-stub	2006-07-31 22:40:00.000000000 +0200
+++ anaconda-11.2.0.24/command-stubs/mknod-stub	2007-02-20 18:20:05.686623000 +0100
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 
 import sys, os
-sys.path.append('/usr/lib/anaconda')
+sys.path.append('/usr/%s/anaconda' % sys.lib)
 import isys
 import string
 import stat
diff -ur anaconda-11.2.0.24/command-stubs/pump-stub anaconda-11.2.0.24/command-stubs/pump-stub
--- anaconda-11.2.0.24/command-stubs/pump-stub	2004-01-23 05:01:21.000000000 +0100
+++ anaconda-11.2.0.24/command-stubs/pump-stub	2007-02-20 18:19:06.622623000 +0100
@@ -7,7 +7,7 @@
 if (os.path.exists('isys')):
     sys.path.append('isys')
 
-sys.path.append('/usr/lib/anaconda')
+sys.path.append('/usr/%s/anaconda' % sys.lib)
 
 import isys
 from sys import argv
diff -ur anaconda-11.2.0.24/command-stubs/raidstart-stub anaconda-11.2.0.24/command-stubs/raidstart-stub
--- anaconda-11.2.0.24/command-stubs/raidstart-stub	2006-03-28 18:49:14.000000000 +0200
+++ anaconda-11.2.0.24/command-stubs/raidstart-stub	2007-02-20 18:19:36.002623000 +0100
@@ -12,7 +12,7 @@
     sys.path.append('gnome-map')
     sys.path.append('isys')
 
-sys.path.append('/usr/lib/anaconda')
+sys.path.append('/usr/%s/anaconda' % sys.lib)
 
 import anaconda_log
 import fsset
diff -ur anaconda-11.2.0.24/command-stubs/raidstop-stub anaconda-11.2.0.24/command-stubs/raidstop-stub
--- anaconda-11.2.0.24/command-stubs/raidstop-stub	2006-03-28 18:49:14.000000000 +0200
+++ anaconda-11.2.0.24/command-stubs/raidstop-stub	2007-02-20 18:20:21.294623000 +0100
@@ -12,7 +12,7 @@
     sys.path.append('gnome-map')
     sys.path.append('isys')
 
-sys.path.append('/usr/lib/anaconda')
+sys.path.append('/usr/%s/anaconda' % sys.lib)
 
 import anaconda_log
 import fsset
diff -ur anaconda-11.2.0.24/command-stubs/syslogd-stub anaconda-11.2.0.24/command-stubs/syslogd-stub
--- anaconda-11.2.0.24/command-stubs/syslogd-stub	2006-05-01 18:39:15.000000000 +0200
+++ anaconda-11.2.0.24/command-stubs/syslogd-stub	2007-02-20 18:20:40.322623000 +0100
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 
 import sys
-sys.path.append('/usr/lib/anaconda')
+sys.path.append('/usr/%s/anaconda' % sys.lib)
 
 def usage():
     sys.stderr.write("syslogd [root] [output file]")
diff -ur anaconda-11.2.0.24/installclass.py anaconda-11.2.0.24/installclass.py
--- anaconda-11.2.0.24/installclass.py	2007-02-06 17:14:43.000000000 +0100
+++ anaconda-11.2.0.24/installclass.py	2007-02-20 18:09:46.158623000 +0100
@@ -485,7 +485,7 @@
     elif os.access("/tmp/product/installclasses", os.R_OK):
         path = "/tmp/product/installclasses"
     else:
-	path = "/usr/lib/anaconda/installclasses"
+	path = "/usr/%s/anaconda/installclasses" % sys.lib
 
     # append the location of installclasses to the python path so we
     # can import them
diff -ur anaconda-11.2.0.24/language.py anaconda-11.2.0.24/language.py
--- anaconda-11.2.0.24/language.py	2006-10-10 19:18:09.000000000 +0200
+++ anaconda-11.2.0.24/language.py	2007-02-20 18:09:46.158623000 +0100
@@ -21,6 +21,7 @@
 from rhpl.simpleconfig import SimpleConfigFile
 
 import logging
+import sys
 log = logging.getLogger("anaconda")
 
 # Converts a single language into a "language search path". For example,
@@ -57,7 +58,7 @@
             self.current = "en_US.UTF-8"
 
         # English name -> native name mapping
-        search = ('lang-names', '/usr/lib/anaconda/lang-names')
+        search = ('lang-names', ('/usr/%s/anaconda/lang-names' % sys.lib))
         for path in search:
             if os.access(path, os.R_OK):
                 f = open(path, 'r')
@@ -72,7 +73,7 @@
         # nick -> (name, short name, font, keyboard, timezone) mapping
         search = ('lang-table', '/tmp/updates/lang-table',
                   '/mnt/source/RHupdates/lang-table', '/etc/lang-table',
-                  '/usr/lib/anaconda/lang-table')
+                  '/usr/%s/anaconda/lang-table' % sys.lib)
         for path in search:
             if os.access(path, os.R_OK):
                 f = open(path, "r")
diff -ur anaconda-11.2.0.24/loader2/loader.c anaconda-11.2.0.24/loader2/loader.c
--- anaconda-11.2.0.24/loader2/loader.c	2007-02-13 17:52:22.000000000 +0100
+++ anaconda-11.2.0.24/loader2/loader.c	2007-02-20 18:17:47.146623000 +0100
@@ -300,6 +300,8 @@
             setenv("PYTHONSTARTUP", "/tmp/updates/pyrc.py", 1);
         else if (!access("/usr/lib/anaconda-runtime/pyrc.py", R_OK|X_OK))
             setenv("PYTHONSTARTUP", "/usr/lib/anaconda-runtime/pyrc.py", 1);
+	else if (!access("/usr/lib64/anaconda-runtime/pyrc.py", R_OK|X_OK))
+	    setenv("PYTHONSTARTUP", "/usr/lib64/anaconda-runtime/pyrc.py", 1);
         setenv("LD_LIBRARY_PATH", LIBPATH, 1);
         setenv("LANG", "C", 1);
         
diff -ur anaconda-11.2.0.24/scripts/pkgorder anaconda-11.2.0.24/scripts/pkgorder
--- anaconda-11.2.0.24/scripts/pkgorder	2007-01-30 23:26:15.000000000 +0100
+++ anaconda-11.2.0.24/scripts/pkgorder	2007-02-20 18:17:08.462623000 +0100
@@ -18,8 +18,8 @@
 import sys
 import yum
 
-sys.path.append("/usr/lib/anaconda")
-sys.path.append("/usr/lib/booty")
+sys.path.append("/usr/%s/anaconda" % sys.lib)
+sys.path.append("/usr/%s/booty" % sys.lib)
 
 import anaconda_log
 import logging

