--- anaconda-11.0.5/Makefile	2006-05-24 18:14:57.185905429 +0300
+++ anaconda-11.0.5/Makefile	2006-05-24 18:46:24.328179443 +0300
@@ -14,6 +14,8 @@
 endif
 endif
 
+OPTFLAGS = -ggdb
+CFLAGS = $(OPTFLAGS)
 CATALOGS = po/anaconda.pot
 
 PYFILES = $(wildcard *.py)
@@ -24,15 +26,15 @@
 	PYTHONPATH="." python scripts/getlangnames.py > lang-names
 
 mini-wm: mini-wm.c
-	gcc -o mini-wm mini-wm.c `pkg-config gtk+-x11-2.0 --cflags --libs`$(CFLAGS)
+	$(CC) -o mini-wm mini-wm.c `pkg-config gtk+-x11-2.0 --cflags --libs`$(CFLAGS)
 
 xmouse.so: xmouse.c
-	gcc -Wall -o xmouse.o -fPIC -I/usr/X11R6/include -I$(PYTHONINCLUDE) -I $(PYTHONINCLUDE) -c xmouse.c $(CFLAGS)
-	gcc -o xmouse.so -shared xmouse.o -L/usr/X11R6/$(LIBDIR) -lXxf86misc -lX11 -lXext
+	$(CC) -Wall -o xmouse.o -fPIC -I/usr/X11R6/include -I$(PYTHONINCLUDE) -I $(PYTHONINCLUDE) -c xmouse.c $(CFLAGS)
+	$(CC) -o xmouse.so -shared xmouse.o -L/usr/X11R6/$(LIBDIR) -lXxf86misc -lX11 -lXext
 
 xutils.so: xutils.c
-	gcc -ggdb -Wall -o xutils.o -fPIC -I/usr/X11R6/include -I$(PYTHONINCLUDE) -I $(PYTHONINCLUDE) -c xutils.c $(CFLAGS)
-	gcc -o xutils.so -shared xutils.o -ggdb -L/usr/X11R6/$(LIBDIR) -lX11
+	$(CC) $(OPTFLAGS) -Wall -o xutils.o -fPIC -I/usr/X11R6/include -I$(PYTHONINCLUDE) -I $(PYTHONINCLUDE) -c xutils.c $(CFLAGS)
+	$(CC) -o xutils.so -shared xutils.o $(OPTFLAGS) -L/usr/X11R6/$(LIBDIR) -lX11
 
 depend:
 	rm -f *.o *.so *.pyc
--- anaconda-11.0.5/isomd5sum/Makefile~	2005-03-02 04:29:39.000000000 +0200
+++ anaconda-11.0.5/isomd5sum/Makefile	2006-05-24 18:37:29.636215985 +0300
@@ -1,6 +1,7 @@
 include ../Makefile.inc
 
-CFLAGS = -g -fPIC -D_FILE_OFFSET_BITS=64 -I$(PYTHONINCLUDE) -O -Wall -Werror \
+OPTFLAGS = -g -O
+CFLAGS = $(OPTFLAGS) -fPIC -D_FILE_OFFSET_BITS=64 -I$(PYTHONINCLUDE) -Wall -Werror \
 	-D_FORTIFY_SOURCE=2
 OBJECTS = md5.o libimplantisomd5.o checkisomd5.o pyisomd5sum.c \
 	  implantisomd5 checkisomd5
@@ -18,14 +19,14 @@
 all: implantisomd5 checkisomd5 pyisomd5sum.so	
 
 %.o: %.c
-	gcc -c -O $(CFLAGS) -o $@ $<
+	$(CC) -c $(OPTFLAGS) $(CFLAGS) -o $@ $<
 
 implantisomd5: implantisomd5.o md5.o libimplantisomd5.o
 
 checkisomd5: checkisomd5.o md5.o libcheckisomd5.o
 
 pyisomd5sum.so: $(PYOBJS)
-	gcc -shared -g -o pyisomd5sum.so -fpic $(PYOBJS)
+	$(CC) -shared $(OPTFLAGS) -o pyisomd5sum.so -fpic $(PYOBJS)
 
 install:
 	install -m 755 implantisomd5 $(DESTDIR)/$(RUNTIMEDIR)
--- anaconda-11.0.5/stubs/Makefile~	2006-02-22 22:00:32.000000000 +0200
+++ anaconda-11.0.5/stubs/Makefile	2006-05-24 18:38:42.777854857 +0300
@@ -9,7 +9,7 @@
 all:	$(TARGETS) libunicode-lite.a
 
 $(LIBUTF8): unicode-lite.c
-	gcc -fPIC -shared -o $@ $< -D_FORTIFY_SOURCE=2
+	$(CC) -fPIC -shared -o $@ $< -D_FORTIFY_SOURCE=2 $(OPTFLAGS)
 
 libunicode-lite.a: libunicode-lite.a($(OBJECTS))
 
--- anaconda-11.0.5/loader2/Makefile~	2006-02-22 22:00:32.000000000 +0200
+++ anaconda-11.0.5/loader2/Makefile	2006-05-24 18:41:33.111671407 +0300
@@ -34,12 +34,13 @@
 HWLIBS += -lpci
 endif
 
-CFLAGS += -DUSE_LOGDEV -DVERSION='"$(VERSION)"'
+OPTFLAGS = -ggdb 
+CFLAGS += -DUSE_LOGDEV -DVERSION='"$(VERSION)"' $(OPTFLAGS)
 STATIC = -static
 REALCC=gcc
 
 ifeq (1, $(USEDIET))
-CFLAGS += -DGZLIB=1 -DDIET=1
+CFLAGS += -DGZLIB=1 -DDIET=1 $(OPTFLAGS)
 DIET=diet
 CC=$(DIET) $(REALCC)
 ISYSLIB = ../isys/libisys-diet.a
@@ -128,7 +129,7 @@
 	$(CC) -DTESTING $(CFLAGS) -o mediacheck.o -c mediacheck.c
 	$(CC) -DTESTING $(CFLAGS) -o log.o -c log.c
 	$(CC) -DTESTING $(CFLAGS) -o md5.o -c md5.c
-	$(CC) -g $(STATIC) -o checkisomd5 mediacheck.o log.o md5.o \
+	$(CC) $(OPTFLAGS) -g $(STATIC) -o checkisomd5 mediacheck.o log.o md5.o \
 		$(NEWTLIB) $(SLANGLIB)
 
 install: all
@@ -142,7 +143,7 @@
 	install -m 644 module-info $(DESTDIR)/$(RUNTIMEDIR)/loader
 
 dirbrowser: dirbrowser.c
-	gcc -DSTANDALONE -D_FORTIFY_SOURCE=2 -Wall -Werror -ggdb -o dirbrowser dirbrowser.c -lnewt -lslang
+	$(CC) -DSTANDALONE -D_FORTIFY_SOURCE=2 -Wall -Werror $(OPTFLAGS) -o dirbrowser dirbrowser.c -lnewt -lslang
 
 ifeq (.depend,$(wildcard .depend))
 include .depend
--- anaconda-11.0.5/isys/Makefile~	2006-02-22 22:04:46.000000000 +0200
+++ anaconda-11.0.5/isys/Makefile	2006-05-24 18:43:02.223663837 +0300
@@ -1,6 +1,7 @@
 include ../Makefile.inc
 
-CFLAGS +=  -I$(PYTHONINCLUDE) -I.. -DHAVE_NFS
+OPTFLAGS = -g
+CFLAGS +=  -I$(PYTHONINCLUDE) -I.. -DHAVE_NFS $(OPTFLAGS)
 
 OBJECTS = nfsmount.o nfsmount_clnt.o nfsmount_xdr.o imount.o getmacaddr.o \
           smp.o devnodes.o cpio.o uncpio.o dasd.o \
@@ -48,10 +49,10 @@
 	$(DIET) $(CC) -c $(CFLAGS) -DGZLIB=1 -DDIET=1 "-Du_char=unsigned char" -o $@ $<
 
 _isys.so: isys.lo $(SOBJECTS)
-	gcc -shared -g -fPIC -o $@ isys.lo $(SOBJECTS) $(LOADLIBES)
+	$(CC) -shared $(OPTFLAGS) -fPIC -o $@ isys.lo $(SOBJECTS) $(LOADLIBES)
 
 _silo.so: silo.c
-	gcc -shared $(CFLAGS) -fpic -o $@ silo.c ../balkan/libbalkan.a
+	$(CC) -shared $(CFLAGS) -fpic -o $@ silo.c ../balkan/libbalkan.a
 
 libisys.a: libisys.a($(OBJECTS))
 
--- anaconda-11.0.5/isys/gzlib/Makefile~	2005-06-13 22:09:23.000000000 +0300
+++ anaconda-11.0.5/isys/gzlib/Makefile	2006-05-24 18:43:33.684366732 +0300
@@ -1,6 +1,6 @@
 include ../../Makefile.inc
 
-CFLAGS += -I . -fPIC
+CFLAGS += -I . -fPIC $(OPTFLAGS)
 SOURCES = bits.c gzip.c inflate.c lzw.c trees.c unzip.c util.c zip.c binding.c \
 	  deflate.c zip.c
 OBJS = $(patsubst %.c,%.o,$(SOURCES))
--- anaconda-11.0.5/utils/Makefile~	2006-01-30 21:40:06.000000000 +0200
+++ anaconda-11.0.5/utils/Makefile	2006-05-24 18:44:44.615951525 +0300
@@ -7,7 +7,8 @@
 
 LOADERDIR = ../loader2
 LOADLIBES =  -lpopt
-CFLAGS += -I.. -I$(LOADERDIR) -fno-strict-aliasing
+OPTFLAGS = -O -g
+CFLAGS += -I.. -I$(LOADERDIR) -fno-strict-aliasing $(OPTFLAGS)
 RPMCFLAGS = $(CFLAGS) -I/usr/include/rpm
 LDFLAGS = -g
 
@@ -37,7 +38,7 @@
             $(LOADLIBES) -lresolv -lpopt -lbz2
 
 md5.o: md5.c md5.h
-	gcc -c -O -g md5.c -D_FORTIFY_SOURCE=2
+	$(CC) -c $(OPTFLAGS) md5.c -D_FORTIFY_SOURCE=2
 
 hash.o : hash.c
 	$(CC) $(RPMCFLAGS) -c -o $@ $<
