--- anaconda-11.0.5/product.py~	2006-03-13 21:12:36.000000000 +0200
+++ anaconda-11.0.5/product.py	2006-05-25 00:21:19.708387579 +0300
@@ -23,10 +23,10 @@
     path = None
     
 
-productName = "anaconda"
-productVersion = "bluesky"
-productPath = "anaconda"
-bugUrl = "your distribution provided bug reporting tool."
+productName = "PLD Linux Distribution"
+productVersion = "2.0 (Ac)"
+productPath = "PLD"
+bugUrl = "http://bugs.pld-linux.org/"
 
 if path is not None:
     f = open(path, "r")
--- anaconda-11.0.5/iutil.py~	2006-06-06 20:14:33.538984888 +0300
+++ anaconda-11.0.5/iutil.py	2006-06-06 20:31:10.801311289 +0300
@@ -656,7 +656,7 @@
     rhpl.arch.canonArch = myarch            
         
     f = open("%s/etc/rpm/platform" %(root,), 'w+')
-    f.write("%s-redhat-linux\n" %(myarch,))
+    f.write("%s-pld-linux\n" %(myarch,))
     f.close()
 
     # FIXME: writing /etc/rpm/macros feels wrong somehow
