--- anaconda-22.4/pyanaconda/packaging/yumpayload.py~	2014-09-18 12:44:59.000000000 +0300
+++ anaconda-22.4/pyanaconda/packaging/yumpayload.py	2014-09-18 12:45:35.407822873 +0300
@@ -1221,7 +1221,7 @@
         if self.data.packages.nocore:
             log.info("skipping core group due to %%packages --nocore; system may not be complete")
         else:
-            self._selectYumGroup("core")
+            self._selectYumGroup("base")
 
         env = None
 
